src/compiler/proposals-and-stabilization.md

```diff
@@ -263,11 +263,25 @@ circumstance.
     in the description
   - New targets should be assigned to the compiler team co-leads to ensure that any demands on
     the project infrastructure are considered and checked with relevant teams
+- Renaming a target or making a breaking change to a tier three target
+  - **Propose using:** PR
+  - **Approve using:** r+
+  - Open an PR with the proposed rename and describe the motivation for the change and obtain a r+
+    from the reviewer.
```

Comment:
Adding a T3 target now needs an ack from a compiler lead, right? I think that was discussed recently.

Reply:
I blieve this is already mentioned below or above

Reply:
You're right, I missed that this is preexisting.